Dockerfile

```diff
@@ -28,14 +30,14 @@ FROM builder AS manager-build
 RUN \
     --mount=type=cache,target=${GOCACHE} \
     --mount=type=cache,target=/go/pkg/mod \
-    GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o out/manager main.go
+    GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -ldflags ${LDFLAGS} -o out/manager main.go
```

Comment:
if the `LDFLAGS` arg is not supplied then build will fail because this expands to `-ldflags   -o...` . Better:
```
GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build ${LDFLAGS:+-ldflags "$LDFLAGS"} -o out/manager main.go
```
`:+` expands to nothing when the variable is undefined or empty.